tasks/utilities/data_export/task_export_taxon_table.wdl

```diff
@@ -72,4 +72,86 @@ task export_taxon_table {
     disk: disk_size + " GB"
     preemptible: 1
   }
+}
+
+task export_taxon_table_vsp {
```

Comment:
these two tasks should be combined as any vsp-specific functionality can be blocked behind a boolean. reduces duplicate code significantly